modules/virt-about-pci-passthrough.adoc

```diff
@@ -3,8 +3,8 @@
 // * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
 
 [id="virt-about_pci-passthrough_{context}"]
-= About preparing nodes and the cluster for PCI passthrough
+= About preparing a host device for PCI passthrough
 
-The Peripheral Component Interconnect (PCI) passthrough feature enables you to access and manage hardware devices from a virtual machine. When PCI passthrough is configured, the PCI devices function as if they were physically attached to the virtual machine.
+To prepare a host device for PCI passthrough using the CLI, create a `MachineConfig` object and add kernel arguments to enable the Input-Output Memory Management Unit (IOMMU). Bind the PCI device to the Virtual Function I/O (VFIO) driver and then expose it in the cluster by editing the `permittedHostDevices` field of the `HyperConverged` custom resource (CR). The `permittedHostDevices` list is empty when you first install the HyperConverged Cluster Operator.
```

Comment:
This is very much a suggestion, but maybe:

s/HyperConverged Cluster Operator/`{VirtProductName}` Operator

The install docs tend to avoid the former: https://docs.openshift.com/container-platform/4.8/virt/install/installing-virt-web.html